### ISSUE TITLE
RSS Podcasts will not display any episodes nor download latest suddenly

### ISSUE THREAD

**Issue by Timaeus1138** [LibreTime/libretime#467]
station has been running flawlessly for almost 3 months, everything downloads, schedules, and plays right on target. 
Randomly now latest episodes not downloaded, and when you go to a podcast you see no content, just the spinning loading image, and the background text "This podcast doesn't have any episodes!".

still running alpha 3, as the station was running fine and i wanted to clone the vm and run the upgrade on the clone prior to full production upgrade.

I've started poking around the logs, but didnt see any major red flags YET. searching the bugs section i've seen other rss issues, but nothing that has led me to the culprit yet. 
Any suggestions where I should start looking?

**Comment by Timaeus1138** [comment]
could the recent Daylight savings time adjustment have something to do with the sudden disappearance of podcast downloads and episodes?

**Comment by Timaeus1138** [comment]
on further investigation, my existing podcasts, if i try to add them again or add to my second copy of libretime show up as invalid feeds, i can however add a npr feed.

example feed that was working that is now "invalid"
http://wildfireradio.com/category/wildfire-originals/classic-rock-throwdown/feed/

working npr feed:
https://www.npr.org/rss/podcast.php?id=510289

**Comment by Robbt** [comment]
First question, did you upgrade and change the code or did it just start not working ?

I'd check the logs under /var/log/airtime/ 
There most likely isn't a /var/log/airtime/airtime-celery.log because we need to make some modification to have this log start (there is an open issue for this) - but you can manually start it with sudo -u www-data airtime-celery start and it will display the messages directly to the console.

Try to add various podcasts and see if you notice an error for the ones that aren't working.

**Comment by Timaeus1138** [comment]
I actually just resolved this and was about to update the thread.

It was not a Libretime issue at all. It was the ISP, the card on file expired and they we're sending  outbound http traffic to a walled garden. But allowing https traffic, inbound connections and pings to http addresses.

It's a headless system on a dedicated ISP network. With no standard desktops/laptops on it.

I did an apt-get update with connection failures then tried some wget commands that failed. Then I connected a standard laptop to the network and got the walled garden page in browser from our ISP. Quick call and it was fixed in 15 mins.

Sorry for the false alarm but thank you for the help!

RSS and downloads have resumed as normal now.
